roles/cifmw_cephadm/tasks/glance_s3_info.yaml

```diff
@@ -0,0 +1,124 @@
+---
+ # Copyright 2025 Red Hat, Inc.
+ # All Rights Reserved.
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License"); you may
+ # not use this file except in compliance with the License. You may obtain
+ # a copy of the License at
+ #
+ #     http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ # License for the specific language governing permissions and limitations
+ # under the License.
+
+- name: Create glance S3 RGW user and fetch object-store endpoint
+  when: cifmw_cephadm_rgw_s3_glance | default(false) | bool
+  block:
+    - name: Refresh ceph_cli
+      ansible.builtin.include_tasks: ceph_cli.yml
+      vars:
+        ceph_command: "radosgw-admin"
+
+    - name: Wait for RGW daemons to be running
+      ansible.builtin.command:
+        cmd: "{{ cifmw_cephadm_ceph_cli }} orch ps --daemon-type rgw --format json"
+      register: rgw_daemon_status
+      become: true
+      changed_when: false
+      failed_when: false
+      until: >
+        rgw_daemon_status.rc == 0 and
+        (rgw_daemon_status.stdout | default('[]') | from_json | default([], true)) |
+        selectattr('status_desc', 'equalto', 'running') |
+        list | length > 0
+      retries: 30
+      delay: 10
+
+    - name: Check if Ceph S3 glance user exists
+      ansible.builtin.command:
+        cmd: >-
+          {{ cifmw_cephadm_ceph_cli }} user info --uid glance
+      no_log: true
+      become: true
+      ignore_errors: true
+      changed_when: false
+      register: cifmw_ceph_s3_glance_user_check
+      when: cifmw_cephadm_ceph_cli is defined
+
+    - name: Create Ceph S3 glance user
+      ansible.builtin.command:
+        cmd: >-
+          {{ cifmw_cephadm_ceph_cli }} user create
+          --uid="glance"
+          --display-name="Glance S3 User"
+      become: true
+      when:
+        - cifmw_ceph_s3_glance_user_check is not skipped
```

Comment:
the condition for that tasks are almost fine. What is missing: keep logic.
In previous task you check `when: cifmw_cephadm_ceph_cli is defined`.
Here you use that binary, so if previous task fails, it is skipped, rc != 0, then if `cifmw_cephadm_ceph_cli` is not defined, it would raise an error.
Do I understand correctly? I'm not familiar with "skipping" in condition and when it is ignore_errors.